engines/build.info

```diff
@@ -32,7 +32,9 @@ IF[{- !$disabled{"engine"} -}]
       MODULES{engine}=padlock
       SOURCE[padlock]=e_padlock.c $PADLOCKASM
       DEFINE[padlock]=$PADLOCKDEF
-      DEPEND[padlock]=../libcrypto
+      IF[{- $target =~ m/^linux/ -}]
+        DEPEND[padlock]=../libcrypto
```

Comment:
what is this change supposed to do?

Reply:
It stops adding `libcrypto.so` as link dependency on `linux`. All other platforms should see no changes.

Reply:
I've found a better way to check for `ld`. I'll try it now and post an update:
```
IF[{- defined $target{shared_defflag} -}]
```